### PR TITLE
Setup and development guides

### PR DESCRIPTION
**Pull request by zamotany**
Ref: #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +65,10 @@ const palette = await MaterialPalette.create(require('./assets/image.jpg'), {
 });
 ```
 
+## Development
+
+Development instructions can be found here: [`react-native-material-palett` development](./docs/DEVELOPEMTN.md).
```
Comment (rgommezz) [inline README.md]: typo `DEVELOPMENT` and `react-native-material-palette-development`

File: docs/DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# `react-native-material-palette` development
+
+The recommended way to develop `react-native-material-palette` is by cloning the repository and symlinkink it to a project using `yarn link` or `npm link`.
```
Comment (rgommezz) [inline docs/DEVELOPMENT.md]: typo symlinking

File: docs/DEVELOPMENT.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+Follow the steps below in order to setup `react-native-material-palette` for development:
+
+1. Clone the repository:
+```bash
+$ git clone https://github.com/callstack-io/react-native-material-palette.git
+```
+
+2. Install dependencies:
+```bash
+$ yarn # or `npm install`
```
Comment (rgommezz) [inline docs/DEVELOPMENT.md:14]: `yarn install` or `npm install`

File: docs/DEVELOPMENT.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+## Using example app
+4. Navigate to the example app:
+```bash
+cd example
+```
+
+5. Install example app dependencies:
+```bash
+$ yarn link # or `npm link`
```
Comment (rgommezz) [inline docs/DEVELOPMENT.md]: This should be:
```bash
yarn install
yarn link react-native-material-palette # or `npm link react-native-material-palette`
```

File: docs/DEVELOPMENT.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+```
+
+Done. Everything is set up aleady. Run `yarn run haul -- --platform andoid` to start the development server and `react-native run-andoid` to build and start the app.
```
Comment (rgommezz) [inline docs/DEVELOPMENT.md]: typo in `android` twice

File: docs/DEVELOPMENT.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+## Custom app
+
+4. (Optional) Create a React Native project and natigate to it:
```
Comment (rgommezz) [inline docs/DEVELOPMENT.md]: typo `navigate`

File: docs/SETUP.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
+# Installation and setup
+
+`react-native-material-palette` is a native module written in Kotlin, thus you need to follow the instructions below in order to set it up.
+
+Once the React Native team releases a new version with the [following change](https://github.com/facebook/react-native/commit/d666f30665e4cd336e1a1017954447f486a3e8ce), you will be able to use `react-native link react-native-material-palette`, but for now you have to do it manually.
```
Comment (rgommezz) [inline docs/SETUP.md:5]: ❤️

File: docs/SETUP.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+## Compatibility
+
+Material Palette SDK is __only available in Android 24 or higher__. We __recommed__ using version __25__.
+
+If you're using a lower version, you need to add
+```
+compile 'com.android.support:support-v4:24.2.0'
+```
+to `andoid/app/src/build.gardle` in `dependencies` section.
```
Comment (rgommezz) [inline docs/SETUP.md]: typo `gradle`

**Review by rgommezz — DISMISSED** [review]

File: docs/DEVELOPMENT.md (hunk 3 above)
Comment (zamotany) [reply docs/DEVELOPMENT.md:14]: `yarn` by default installs dependencies, `yarn install` is just more typing 😛

**Pull request merged**